enacts/flex_fcst/cpt.py

```diff
@@ -106,6 +106,32 @@ def starts_list(
     return start_dates
 
 
+def read_mpycptv2dataset(data_path):
+    mu_mslices = []
+    var_mslices = []
+    obs_slices = []
+    target_count = 0
+    for targets in Path(data_path).iterdir() :
+        target_count = target_count + 1
+    for targets in Path(data_path).iterdir() :
+        new_mu, new_var, new_obs = read_pycptv2dataset(targets)
+        if target_count > 1 :
+            L = (((new_mu["Ti"].dt.month - new_mu["S"].dt.month).squeeze() + 12) % 12).values
+            new_mu = new_mu.assign_coords({"L": L}).expand_dims(dim="L")
+            new_var = new_var.assign_coords({"L": L}).expand_dims(dim="L")
+        mu_mslices.append(new_mu)
+        var_mslices.append(new_var)
+        obs_slices.append(new_obs)
+    fcst_mu = xr.combine_by_coords(mu_mslices)["deterministic"]
+    fcst_var = xr.combine_by_coords(var_mslices)["prediction_error_variance"]
+    if target_count == 1:
+        fcst_mu = fcst_mu.squeeze(L, drop=True)
+        fcst_var = fcst_var.squeeze(L, drop=True)
```

Comment:
I don't understand this. If `target_count == 1` then we didn't add `L` in the first place (line 118),